### PR TITLE
Implement Next action in android IME

### LINKED ISSUE #12758: Mobile device soft keyboard 'Next' button does not shift focus to next input
**Describe the bug**
Creating an Input element with attribute/value `TextInputOptions.ReturnKeyType="Next"` correctly configures the soft keyboard with a 'Next' return key but the key does shift focus to the next input.

**To Reproduce**

1. Create an Avalonia cross-platfom app: `dotnet new avalonia.xplat -o MyApp`
2. Edit the non-boileplate content of `MainView.xaml`:
```
<StackPanel>
    <TextBox TextInputOptions.ReturnKeyType="Next"></TextBox>
    <TextBox TextInputOptions.ReturnKeyType="Next"></TextBox>
</StackPanel>
```
3. Deploy to an Android or iOS emulator
4. Click in first TextBox: soft keyboard is displayed. Select 'Next' or '->|' button: input focus does not change.

**Expected behavior**
Input focus should move to the second TextBox.

**Desktop (please complete the following information):**

- OS: Windows/Android emulator, MacOS/iOS simulator
- Version 11.0.4

### PR DESCRIPTION
**Pull request by emmauss**
## What does the pull request do?
Implements the Next ime action for Android

## Checklist

- [ ] Added unit tests (if possible)?
- [ ] Added XML documentation to any related classes?
- [ ] Consider submitting a PR to https://github.com/AvaloniaUI/avalonia-docs with user documentation

## Fixed issues
Partially fixes https://github.com/AvaloniaUI/Avalonia/issues/12758 . iOS actions has not been implemented.

### CODE REVIEW

**Comment by emmauss** [comment]
@maxkatz6 if you could add the iOS action.

File: src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs [hunk 1]
```diff
@@ -624,6 +624,12 @@ public override bool PerformEditorAction([GeneratedEnum] ImeAction actionCode)
                         _inputMethod.IMM.HideSoftInputFromWindow(_inputMethod.View.WindowToken, HideSoftInputFlags.ImplicitOnly);
                         break;
                     }
+                case ImeAction.Next:
+                    {
+                        var focusedElement = FocusManager.GetFocusManager(_toplevel.InputRoot).GetFocusedElement();
```
Comment (Gillibald) [inline src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs:629]: Can we share this logic somewhere so every platform can call the same logic?

`InputMethod.ExecuteAction(ActionEnum)`

Not sure where this logic should be implemented

Comment (Gillibald) [reply src/Android/Avalonia.Android/Platform/SkiaPlatform/TopLevelImpl.cs:629]: Ideally, someone could register some delegate that gets executed for a given action type

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0040816-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Comment by maxkatz6** [comment]
@emmauss done

**Review by maxkatz6 — APPROVED** [review]

**Review by Gillibald — APPROVED** [review]

**Pull request merged**